core/src/main/groovy/noe/rhel/server/httpd/HttpdBaseOS.groovy

```diff
@@ -0,0 +1,119 @@
+package noe.rhel.server.httpd
+
+import groovy.util.logging.Slf4j
+import noe.common.utils.Cmd
+import noe.common.utils.JBFile
+import noe.common.utils.Library
+import noe.common.utils.Version
+import noe.server.Httpd
+/**
+ * RHEL BaseOS Httpd server
+ * @author Jan Onderka <jonderka@redhat.com> *
+ */
+@Slf4j
+class HttpdBaseOS extends Httpd {
+  protected String serviceName
+
+  HttpdBaseOS(String basedir, version) {
+    super(basedir, version)
+    this.serviceName = "httpd"
+    this.basedir = "/etc/${serviceName}"
+
+    // please initialize in setDefault() method
+    setDefault()
+    logDirs.each { JBFile.makeAccessible(new File(getServerRoot() + it)) }
+  }
+
+  void setDefault() {
+    super.setDefault()
+    this.configDirs = ['/conf', '/conf.d']
+    if (version >= new Version("2.4")) {
+      this.configDirs.add('/conf.modules.d')
+    }
+    this.modulesDir = this.basedir + "/modules"
+    this.logDirs = ['/logs']
+    // what path will be taken as relative for star/stop cmd
+    this.binPath = '/'
+    if (platform.isRHEL6() || platform.isRHEL5() || platform.isRHEL4()) {
```

Comment:
This is something we still need?

Reply:
+1,  we definitely don't need RHEL 5 or 4 (and should delete it from the codebase). We still support RHEL 6, so that is OK imho.

Reply:
next commit